public/manifest.json

```diff
@@ -1,21 +1,16 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Davidia",
+  "name": "Davidia",
   "icons": [
     {
       "src": "favicon.ico",
```

Comment:
You haven't replaced the favicon with https://www.diamond.ac.uk/dam/Images/icons/favicon.ico

Reply:
favicon.ico is a different file with the same name.

Reply:
Ah, yes, I will mislead by the red and missed the 21% reduction. Note this new `.ico` is only 16x16.